rpc/grpc/api.go

```diff
@@ -37,3 +49,346 @@ func (bapi *broadcastAPI) BroadcastTx(ctx context.Context, req *RequestBroadcast
 		},
 	}, nil
 }
+
+type BlockAPI struct {
+	sync.Mutex
+	heightListeners      map[chan NewHeightEvent]struct{}
+	newBlockSubscription eventstypes.Subscription
+}
+
+func NewBlockAPI() *BlockAPI {
+	return &BlockAPI{
+		// TODO(rach-id) make 1000 configurable if there is a need for it
```

Comment:
[optional] omit the `(rach-id)` part and create a GH issue for this follow-up

```suggestion
		// TODO make 1000 configurable if there is a need for it
```

or omit the TODO entirely if there is no need for it yet

```suggestion
```

Reply:
there doesn't seem to be a need for it now. cool will delete